src/openrct2/object/FootpathItemObject.cpp

```diff
@@ -25,7 +25,7 @@ void FootpathItemObject::ReadLegacy(IReadObjectContext* context, OpenRCT2::IStre
     stream->Seek(6, OpenRCT2::STREAM_SEEK_CURRENT);
     _legacyType.path_bit.flags = stream->ReadValue<uint16_t>();
     _legacyType.path_bit.draw_type = stream->ReadValue<uint8_t>();
-    _legacyType.path_bit.tool_id = stream->ReadValue<uint8_t>();
+    _legacyType.path_bit.tool_id = stream->ReadValue<CursorID>();
```

Comment:
can you make it so that all of the ReadValue calls that use CursorID stick to uint8_t. Wrap it in a static cast.
The reason we want to do this is so that if someone changes the type size we don't want to break object loading. In theory we would be validating the value that is read before converting it into a CursorID but...